source/contracts/reporting/Reporting.sol

```diff
@@ -55,4 +60,9 @@ library Reporting {
     function targetDesignatedReportNoShowsDivisor() internal pure returns (uint256) { return TARGET_DESIGNATED_REPORT_NO_SHOWS_DIVISOR; }
     function targetRepMarketCapMultiplier() internal pure returns (uint256) { return TARGET_REP_MARKET_CAP_MULTIPLIER; }
     function forkMigrationPercentageBonusDivisor() internal pure returns (uint256) { return FORK_MIGRATION_PERCENTAGE_BONUS_DIVISOR; }
+    function bondPayoutMultiplier() internal pure returns (uint256) { return BOND_PAYOUT_MULTIPLIER; }
+    function forkRepMigrationVictoryDivisor() internal pure returns (uint256) { return FORK_REP_MIGRATION_VICTORY_DIVISOR; }
+    function maximumReportingFeeDivisor() internal pure returns (uint256) { return MAXIMUM_REPORTING_FEE_DIVISOR; }
+    function defaultReportingFeeDivisor() internal pure returns (uint256) { return DEFAULT_REPORTING_FEE_DIVISOR; }
+    function initialREPSupply() internal pure returns (uint256) { return INITIAL_REP_SUPPLY; }
```

Comment:
For naming consistency, these should _all_ be named `get*` as that is the pattern we use for dumb getters everywhere else.

Alternatively, we should remove the `get` prefix from everything else in the repo.  ;)

Reply:
I'll opt for putting get on these ;p